### DISCUSSION TITLE
Unstable Bluetooth Audio on Fedora 40 with AirPods & JBL Speaker – Need Help!

### DISCUSSION THREAD

**Discussion by unt3rhofer** [i3/i3#6308] [Q&A]
I am encountering significant issues using my **AirPods Pro 2** and my **JBL Bluetooth speaker** with my laptop. Here's my setup:

- **Laptop**: ThinkPad E14 Gen 6  
- **OS**: Fedora 40 with i3 Window Manager  
- **Use case**: I often listen to 4-hour ambiance music videos on YouTube while working or gaming.

Notably, these issues do not occur when I use the laptop’s **built-in speakers**, which leads me to believe the problem lies with the **Bluetooth setup**.

### **Issues with AirPods**:
1. **Crackling Audio**: After a random period, the sound starts crackling, progressively worsening until I restart PipeWire (`systemctl --user restart pipewire`) or reconnect the AirPods.  
2. **Multiple Audio Tracks**: When playing two separate audio tracks simultaneously, the AirPods frequently break. To fix this, I must close all audio-playing windows, restart PipeWire, and reconnect the Bluetooth devices.  
3. **Unpausing Issues**: Sometimes, pausing a video causes audio playback to fail when I unpause it.  
4. **Changing Videos**: Switching to another video often stops audio playback entirely, requiring a PipeWire restart.

### **Issues with JBL Speaker**:
The issues are mostly similar, except:  
- Instead of audio stopping completely, it shifts to the **laptop's built-in speakers**.

For example, during a recent **DnD session** with friends, I was playing one track for background music and another for ambiance. After about an hour, the ambiance noises shifted to the **laptop speakers** instead of the **Bluetooth speaker**. The speaker was plugged in the entire time, so it wasn’t a **battery issue**.

### **Troubleshooting & Forum Suggestions**:
I’m unsure how to troubleshoot this effectively or where to post my questions. If you have recommendations for **forums or subreddits**, I’d greatly appreciate them.

Thank you!

**Comment by okraits** [comment]
I recommend to rather ask in the Fedora community as this isn't related to i3 at all.
